pkg/reconciler/knativeserving/controller.go

```diff
@@ -92,6 +93,31 @@ func NewControllerWithConfig(cfg *rest.Config) func(context.Context, configmap.W
 			Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 		})
 
+		// Reporting statistics on KnativeServing events.
```

Comment:
Does knativeeventing have the following statsReporting?

Reply:
Nope.